package.json

```diff
@@ -60,7 +60,6 @@
     "web-performance"
   ],
   "dependencies": {
-    "js-base64": "^3.6.0",
-    "whatwg-fetch": "^3.5.0"
```

Comment:
Are you implementing `fetch` by ourselves?

Reply:
Are we totally rewriting the thing? And the IP is clear?

Reply:
I just extended functions to the `window.fetch` to intercept http requests with fetch. I think there are no problems of IP.

Reply:
`window.fetch` is the browser's own function.